components/List/tests/snapshot/MDCSelfSizingStereoCellSnapshotTests.m

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialSnapshot.h"
+
+#import "MaterialList.h"
+
+/** Snapshot tests for MDCBaseCell. */
+@interface MDCSelfSizingStereoCellSnapshotTests : MDCSnapshotTestCase
+
+/** The view being tested. */
+@property(nonatomic, strong) MDCSelfSizingStereoCell *cell;
+
+@end
+
+@implementation MDCSelfSizingStereoCellSnapshotTests
+
+- (void)setUp {
+  [super setUp];
+
+  // Uncomment below to recreate all the goldens (or add the following line to the specific
+  // test you wish to recreate the golden for).
+  // self.recordMode = YES;
+
+  self.cell = [[MDCSelfSizingStereoCell alloc] initWithFrame:CGRectMake(0, 0, 240, 60)];
+}
+
+- (void)tearDown {
+  self.cell = nil;
+
+  [super tearDown];
+}
+
+- (void)generateSnapshotAndVerifyForView:(UIView *)view {
+  UIView *snapshotView = [view mdc_addToBackgroundView];
+  [self snapshotVerifyView:snapshotView];
+}
+
+- (void)speedAllLayersOfView:(UIView *)view {
```

Comment:
Looks like this is no longer used - can it be removed?

Reply:
Thanks for catching it. Addressed in c9bb6ef6ce3e3bf619e9e29aa057e22598ebdf9a.